packages/teams-js/test/framelessPostMocks.ts

```diff
@@ -40,7 +40,13 @@ export class FramelessPostMocks {
           return;
         },
       },
-      setInterval: (handler: TimerHandler, timeout?: number, ...args: unknown[]): number =>
+      /* For setInterval, we are intentionally not allowing the TimerHandler type since it allows for either Function or string and string
```

Comment:
Thank you for including this explanation!

Reply:
Yes I think it's good reference for any next time we need to disable a lint rule.